src/tools/common/sss_tools.c

```diff
@@ -333,13 +338,15 @@ errno_t sss_tool_route(int argc, const char **argv,
                 return tool_ctx->init_err;
             }
 
-            ret = tool_cmd_init(tool_ctx, &commands[i]);
-            if (ret != EOK) {
-                DEBUG(SSSDBG_FATAL_FAILURE,
-                      "Command initialization failed [%d] %s\n",
-                      ret, sss_strerror(ret));
-                return ret;
-            }
+	    if (!_help) {
```

Comment:
By convention, we only start the name of a variable with an underscore if the variable is a pointer that is assigned to in the function. This variable is not a pointer, just pass-by-value, so it should be named just `_help`.